react-native/todo/src/components/Todos/components/TodoList/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import {
+  Text,
+  ScrollView,
+  View
+} from 'react-native';
+
+import actionCreators from '../../../../redux/todos/actions';
+
+import Checkbox from './components/Checkbox';
+import DeleteButton from './components/DeleteButton';
+import styles from './styles';
+
+class TodoList extends Component {
+  toggleClickHandler = todo => {
+    this.props.dispatch(actionCreators.toggleTodo(todo));
+  }
+
+  deleteClickHandler = todo => {
+    this.props.dispatch(actionCreators.removeTodo(todo));
+  }
+
+  render() {
+    return (
+      <ScrollView style={styles.todoList}>
+        {
+          this.props.todos && this.props.todos.sort(function(a, b) {
```

Comment:
Notice that we are `sorting` a list every time we `render` this component. Would you like to extract this in order to do that `sort` only once ?